src/Serilog.Sinks.Raygun/Sinks/Raygun/RaygunSink.cs

```diff
@@ -278,5 +281,108 @@ private static RaygunIdentifierMessage ParseUserInformation(string userInfo)
 
             return userIdentifier;
         }
+
+        private static RaygunErrorStackTraceLineMessage[] BuildStackTrace(StackTrace stackTrace)
```

Comment:
It looks like the Raygun4Net version of this code expects an exception and is not public so cannot be re-used without some changes to Raygun4Net first. However the base class `RaygunErrorMessageBuilderBase` is public abstract so you could create a new builder that inherits from `RaygunErrorMessageBuilderBase` which allows you to reuse the `GenerateMethodName` funtion. This might be a good first step, breaking this code out into a separate builder class that inherits from the Raygun4Net base class.

Reply:
I'll take a look at doing this. Next I was planning to update the Raygun4Net client used by this sink to the latest version, so it may be worth updating Raygun4Net to make both these functions reusable from there first anyway.